src/unforge/Program.cs

```diff
@@ -1,30 +1,27 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Threading;
+using System.Globalization;
+using System.Xml;
 
 namespace unforge
 {
-    class Program
+    internal static class Program
```

Comment:
Careful with this - I *might* be referencing this directly from unp4k.gui.

Hopefully not, but FYI

Reply:
I could always make wrapper projects for unp4ck and unforge and change the existing projects into libraries. I mean, it should really be like that. Referencing code cross executables is never a good practise.

Reply:
I think I moved it to a separate class so the entrypoint is no longer `Program`.

I always intended to keep the binaries referenced directly though, the intention of the gui project was to just provide an interface to a pipeline of external tools that you could essentially "pipe" the data streams through. For now, they were all tools I wrote, but the guys working in 3D may be interested in piping through cgfconverter, material converters, etc too in future.

As noted though, that project has been a bit slow to update for a while now. I mostly wanted a cross-platform GUI library to build on, and wasn't particularly happy with the performance of the TreeView I found.

Now that MAUI is on the horizon though, it may be time to pick things back up and see where we can get to.

Reply:
Ye, I was going to mention MAUI. I'm sure everyone will be fine with just WPF for now.

As for other tools, they will still be able to be used. unforged is just a library now which has a wrapper executable console project and can be used by the GUI project. Nothing has really changed from your intention.